src/main/java/Task.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * This class represents a task that can be created
+ * through user command.
+ */
+public class Task {
```

Comment:
Just my opinion: any reason why you chose to use a normal class intead of an abstract class? I notice that you never instantiate a Task object in your main code anyway.